src/graph/LdapAuthenticator.cpp

```diff
@@ -0,0 +1,361 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "graph/LdapAuthenticator.h"
+#include "graph/GraphFlags.h"
+
+namespace nebula {
+namespace graph {
+
+Status LdapAuthenticator::prepare() {
+    if (FLAGS_ldap_server.empty()) {
+        return Status::Error("LDAP authentication \"ldap_server\" parameter is not set.");
+    }
+
+    if (FLAGS_ldap_scheme.compare("ldap")) {
+        return Status::Error("LDAP authentication ldap_scheme only support \"ldap\".");
+    }
+
+    if (FLAGS_ldap_port == 0) {
+        FLAGS_ldap_port = ldapPort_;
```

Comment:
How about just set the default port in define of gflags

Reply:
Yeah，good